frontend/src/pages/CreateCard.js

```diff
@@ -1,33 +1,31 @@
-import React, { useState, Component } from "react";
+import React, { useState } from "react";
 import Header from "../main-components/Header";
 import Sidebar from "../main-components/Sidebar";
-import styled from "@emotion/styled";
 import CreateCardContent from "../main-components/CreateCardContent";
+import queryString from "query-string";
 
-function CreateCard() {
+function CreateCard(props) {
   // Handles mobile menu button and updates sidebar view
-  const [sideSetting, setSideSetting] = useState("f");
+  const [sideBarVisibility, setSideBarVisibility] = useState(false);
   const handleClick = (e) => {
-    console.log("Clicked");
-    if (sideSetting === "f") {
-      setSideSetting("t");
-    } else {
-      setSideSetting("f");
-    }
+    setSideBarVisibility((sideBarVisibility) => !sideBarVisibility);
```

Comment:
as mentioned in another PR, the logic to hide the sidebar is repeated numerous times. Would be good to abstract this out.

Reply:
attempted in another PR.